src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -105,35 +106,47 @@ public function configure(array $configuration = null)
     {
         parent::configure($configuration);
 
-        $this->configuration = self::$defaultConfiguration;
+        $this->configuration = [];
 
         if (0 === count($configuration)) {
+            $this->resolveTokenDefaults(self::SINGLE_SPACE);
+
             return;
         }
 
-        foreach ($configuration as $name => $value) {
-            if ('align_equals' === $name || 'align_double_arrow' === $name) {
-                $this->configuration = $this->resolveOldConfig($configuration);
+        if (isset($configuration['default'])) {
+            $this->resolveTokenDefaults($configuration['default']);
+        } else {
+            $this->resolveTokenDefaults(self::SINGLE_SPACE);
+        }
 
-                break;
-            }
+        if (isset($configuration['align_equals']) || isset($configuration['align_double_arrow'])) {
+            $this->resolveOldConfig($configuration);
```

Comment:
is there a deprecation triggered when one uses old config ?

Reply:
yes, 
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2573/files#diff-fdd74d10fe54a085bcd3669c08cb37d9R399
tested here:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2573/files#diff-09e3cb60a7a2729fabc2c9950e007302R665

Reply:
:+1:
sorry, github linked me in notification only to latest commit, in which i couldn't find it.
thanks